# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit/MIROps.yaml
# Commit: 349728105c40
# Full Hash: 349728105c40993896170f4c2f43e152a3466706
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-01-08 09:40:24
# Description:
#   Bug 1748602 - Backout bug 1642412. r=lth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D135346
# ==============================================================================

diff -r 32873022d2ed -r 349728105c40 js/src/jit/MIROps.yaml
--- a/js/src/jit/MIROps.yaml	Fri Jan 07 20:35:15 2022 +0000
+++ b/js/src/jit/MIROps.yaml	Fri Jan 07 20:43:16 2022 +0000
@@ -2605,9 +2605,6 @@
 - name: WasmLoadGlobalCell
   gen_boilerplate: false
 
-- name: WasmLoadTableElement
-  gen_boilerplate: false
-
 - name: WasmStoreGlobalVar
   gen_boilerplate: false
 
@@ -2620,9 +2617,6 @@
 - name: WasmDerivedPointer
   gen_boilerplate: false
 
-- name: WasmDerivedIndexPointer
-  gen_boilerplate: false
-
 - name: WasmStoreRef
   gen_boilerplate: false
 